src/renderer/components/FtSlider/FtSlider.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <label
+    class="pure-material-slider"
+    :for="id"
+  >
+    <input
+      :id="id"
+      v-model.number="currentValue"
+      :disabled="disabled"
+      type="range"
+      :min="minValue"
+      :max="maxValue"
+      :step="step"
+      @change="change"
+    >
+    <span>
+      {{ t('Display Label', {label: label, value: displayLabel}) }}
+    </span>
+  </label>
+</template>
+
+<script setup>
+import { computed, ref } from 'vue'
+import { randomUUID } from 'crypto'
```

Comment:
I think useId should be used instead

https://github.com/FreeTubeApp/FreeTube/blob/development/src/renderer/composables/use-id-polyfill.js